docs/core-protocol/smart-contracts-api/PublicLock.md

```diff
@@ -1612,37 +1565,23 @@ A function which lets a Lock manager of the lock to change the price for future
 | _keyPrice | uint256 | The new price to set for keys |
 | _tokenAddress | address | The address of the erc20 token to use for pricing the keys, or 0 to use ETH |
 
-### updateLockName
-
-```solidity
-function updateLockName(string _lockName) external nonpayable
-```
-
-Allows a Lock manager to assign a descriptive name for this Lock.
-
-*Throws if called by other than a Lock manager*
-
-#### Parameters
-
-| Name | Type | Description |
-|---|---|---|
-| _lockName | string | The new name for the lock |
-
-### updateLockSymbol
+### updateLockConfig
 
 ```solidity
-function updateLockSymbol(string _lockSymbol) external nonpayable
+function updateLockConfig(uint256 _newExpirationDuration, uint256 _maxNumberOfKeys, uint256 _maxKeysPerAcccount) external nonpayable
 ```
 
-Allows a Lock manager to assign a Symbol for this Lock.
+Update the main key properties for the entire lock:  - default duration of each key - the maximum number of keys the lock can edit - the maximum number of keys a single address can holdkeys previously bought are unaffected by this changes in expiration duration (i.e. existing keys timestamps are not recalculated/updated)
```

Comment:
typo - "can holdkeys"

Reply:
I think we can format that better @clemsos !